configs/models/unet/config_train_unet_with_mobious.yaml

```diff
@@ -18,6 +18,11 @@ transforms: # Transformations are defined in train_mobious.py using transforms.C
     transform_operation: None # Options are 'transforms_img', 'transforms_rotations', None
     target_transform_operation: None # Options are 'transforms_img', 'transforms_rotations', None
 
+datasets_proportions: # Ensure sum of all proportions = 1.
```

Comment:
I believe dataset_splitting_ratios is commonly used terminology, even though 'proportions' means the same thing. So just rename datasets_proportions with `dataset_splitting_ratios`